sample/kotlin/src/main/kotlin/com/datadog/android/sample/widget/WidgetIntentService.kt

```diff
@@ -31,19 +31,17 @@ class WidgetIntentService : IntentService("WidgetIntentService") {
                 val hasRumContext = widgetId != 0 && widgetName != null
 
                 if (hasRumContext) {
-                    GlobalRum.get().startView(widgetId, widgetName ?: "DatadogWidget", emptyMap())
                     val clickedTargetName = intent.getStringExtra(WIDGET_CLICKED_TARGET_NAME)
                     if (clickedTargetName != null) {
-                        GlobalRum.get()
-                            .addUserAction(RumActionType.CLICK, clickedTargetName, emptyMap())
+                        GlobalRum.get().addUserAction(
```

Comment:
So this will be reported with a background scope?

Reply:
This was a test I made, I'll revert that